pkg/bloomcompactor/controller.go

```diff
@@ -67,14 +67,15 @@ Compaction works as follows, split across many functions for clarity:
 func (s *SimpleBloomController) compactTenant(
 	ctx context.Context,
 	table config.DayTime,
+	period config.PeriodConfig,
 	tenant string,
 	ownershipRange v1.FingerprintBounds,
 ) error {
-	logger := log.With(s.logger, "ownership", ownershipRange, "org_id", tenant, "table", table)
```

Comment:
You can change the signature to accept the struct, not a pointer.

Reply:
I don't think we need to log the full table name, the day is probably more convenient.

Reply:
I'd say seeing the full table name is clearer wrt which TSDB table are we working with (and where will we write to)